snap/snapcraft.yaml

```diff
@@ -167,6 +169,14 @@ plugs:
   shutdown: null
   ssh-keys: null
   system-observe: null
+  shell-session-locale-files:
+    interface: personal-files
+    read:
```

Comment:
I think read is included with write access

Reply:
You are right... fixing it.

Reply:
Done.